tutorials/scripting/gdscript/gdscript_exports.rst

```diff
@@ -268,24 +268,23 @@ Exported arrays can specify type (using the same hints as before).
 
 ::
 
-    @export(Array, int) var ints = [1, 2, 3]
-    @export(Array, int, "Red", "Green", "Blue") var enums = [2, 1, 0]
-    @export(Array, Array, float) var two_dimensional = [[1.0, 2.0], [3.0, 4.0]]
+    @export var ints: Array[int] = [1, 2, 3]
+    @export var two_dimensional: Array[Array[float]] = [[1.0, 2.0], [3.0, 4.0]]
 
 You can omit the default value, but it would then be ``null`` if not assigned.
 
 ::
 
-    @export(Array) var b
-    @export(Array, PackedScene) var scenes
+    @export var b: Array
+    @export var scenes: Array[PackedScene]
 
 Arrays with specified types which inherit from resource can be set by
 drag-and-dropping multiple files from the FileSystem dock.
 
 ::
 
-    @export(Array, Texture) var textures
-    @export(Array, PackedScene) var scenes
+    @export var textures: Array[Texture] = []
+    @export var scenes: Array[PackedScene] = []
```

Comment:
I initialized those to an empty array as it's probably good practice to do so most of the time. The example above tells you that initialization is not mandatory. However, it's probably a good idea to mention that initializing arrays is often better than not initializing them to avoid run-time errors.